aimsAndScope.md

```diff
@@ -1,10 +1,22 @@
 # Aims and scope
 
-*Journal of Cheminformatics* is an open access journal publishing original peer-reviewed research in all
+*Journal of Cheminformatics* is an open science journal publishing original peer-reviewed research in all
 aspects of cheminformatics and molecular modelling.
 
 Coverage includes, but is not limited to:
 
 * chemical information systems, software and databases, and molecular modelling
 * chemical structure representations and their use in structure, substructure, and similarity searching of chemical substance and chemical reaction databases
 * computer and molecular graphics, computer-aided molecular design, expert systems, QSAR, and data mining techniques
+
+## Open Science
+
+The journal has the following open science ambitions:
+
+* publish research or software that is entirely reproducible by third parties
+* access to any datasets, software and algorithms that are required to reach the conclusions
```

Comment:
The `software and algorithms` is similarly problematic. What happens if the algorithm that is used in MOE, JChem etc